backend/api/postService.js

```diff
@@ -1,3 +1,20 @@
-module.exports = function (app) {
+const postModel = require("../models/post");
+const { authUserLoggedIn } = require("../middlewares/acl");
 
-}
\ No newline at end of file
+module.exports = function (app) {
+  app.post("/api/user/posts", authUserLoggedIn, async (req, res) => {
+    try {
```

Comment:
Because we are gonna make tests on routes, isn't it better to use if-statements for more accurate tests?
for example if user has not provided req.body : if (!req.body) {res.sendstatus(403); return;} 
then when you are saving object try const res = await newPost.save(); 
then (!res) { res.sendstatus(400); return;} and so on.